e2e/js_run_devserver/MODULE.bazel

```diff
@@ -14,6 +14,13 @@ local_path_override(
     path = "../..",
 )
 
+archive_override(
```

Comment:
close to a rnjs release so we don't need these?

Reply:
yeah... I'm just waiting on a few more PRs to alnd and I'll cut 6.1.0 and update this PR